client/interledger/js/components/account_detail.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import classnames from 'classnames';
+import { Col } from 'react-bootstrap/lib';
+
+import Assets from './assets';
+
+
+const AccountDetail = React.createClass({
+    propTypes: {
+        account: React.PropTypes.object,
+        activeAccount: React.PropTypes.object,
+        assetList: React.PropTypes.object,
+        handleClick: React.PropTypes.func
+    },
+
+    render() {
+        const {
+            account,
+            activeAccount,
+            assetList,
+            handleClick
+        } = this.props;
+        
+        if (account) {
```

Comment:
Don't really have a preference, but since React 15 allows you to return null from stateless components, I've been meaning to convert some of the helper components to return null.